backends/advanced/src/advanced_omi_backend/services/transcription/__init__.py

```
@@ -42,67 +45,43 @@ def get_transcription_provider(
+                elif provider == "parakeet":
+                    service_url = str(_resolve_value(stt_def.get("model_url", "")))
+                    if not service_url:
+                        logger.warning("Parakeet selected in config.yml but model_url missing or empty")
+                        return None
+                    return ParakeetStreamingProvider(service_url) if mode == "streaming" else ParakeetProvider(service_url)
+                else:
+                    logger.warning(f"Unsupported STT provider in config.yml: {provider}")
+                    return None
+    except Exception as e:
+        logger.warning(f"Failed to load STT provider from config.yml: {e}")
+        raise e
```

Ruff (0.14.8)

Use `raise` without specifying exception name

Remove exception name

(TRY201)